File: src/TaskOTime/TaskOTime.ViewModel/ViewModels/TimeEntryCreatedEventArgs.vb
Namespace ViewModels
    Public Class TimeEntryCreatedEventArgs
        Inherits EventArgs

        Public Sub New(entryTime As DateTime, completeRunningTask As Boolean)
            Me.EntryTime = entryTime
            Me.CompleteRunningTask = completeRunningTask
        End Sub

        Public ReadOnly Property EntryTime As DateTime
        Public ReadOnly Property CompleteRunningTask As Boolean
    End Class
End Namespace

File: src/TaskOTime/TaskOTime.ViewModel/ViewModels/TimeEntryCreatedEventArgs.cs
using System;

namespace TaskOTime.ViewModel.ViewModels
{
    public class TimeEntryCreatedEventArgs : EventArgs
    {

        public TimeEntryCreatedEventArgs(DateTime entryTime, bool completeRunningTask)
        {
            EntryTime = entryTime;
            CompleteRunningTask = completeRunningTask;
        }

        public DateTime EntryTime { get; private set; }
        public bool CompleteRunningTask { get; private set; }
    }
}